roles/pulp_database/vars/RedHat-7.yml

```diff
@@ -6,3 +6,4 @@ postgresql_bin_path: /opt/rh/rh-postgresql96/root/bin
 postgresql_data_dir: /var/opt/rh/rh-postgresql96/lib/pgsql/data
 postgresql_config_path: /var/opt/rh/rh-postgresql96/lib/pgsql/data
 postgresql_profile_path: /etc/profile.d/enablepostgresql96.sh
+postgresql_python_library: python3-psycopg2
```

Comment:
I think this is incorrect because our ansible_python_interpreter is python2 on EL7. If you think it is python3, then we need to figure out why it is, and should it be.

It looks like it this line is unnecessary too:
https://github.com/geerlingguy/ansible-role-postgresql/blob/master/defaults/main.yml#L9